cmd/policy-assistant/pkg/matcher/explain.go

```diff
@@ -50,7 +50,7 @@ func (s *SliceBuilder) TargetsTableLines(targets []*Target, isIngress bool) {
 		ruleType = "Egress"
 	}
 	for _, target := range targets {
-		sourceRules := slice.Sort(target.SourceRules)
```

Comment:
I recommend sorting all output so that it's:

 - deterministic
 - predictable for the reader